packages/widget/src/controllers/wallet-manager/token-balance.ts

```diff
@@ -0,0 +1,108 @@
+import type { EvmResource, Resource } from '@buildwithsygma/sygma-sdk-core';
+import { ResourceType } from '@buildwithsygma/sygma-sdk-core';
+import { Web3Provider } from '@ethersproject/providers';
+import { ContextConsumer } from '@lit/context';
+import { BigNumber, Contract } from 'ethers';
+import type { ReactiveController, ReactiveElement } from 'lit';
+import type { ERC20 } from '@buildwithsygma/sygma-contracts';
+import { walletContext } from '../../context';
+import { isEvmResource } from '../../utils';
+import { ierc20Abi } from './abi/IERC20';
```

Comment:
I think you can get from `@buildwithsygma/sygma-contracts` dependency the `IERC20__factory` from which you can get the abi for ERC20 like, `IERC20__factory.abi`. With that you can remove the small ABI that you defined above.